tower/src/retry/future.rs

```diff
@@ -63,7 +63,7 @@ where
 
 impl<P, S, Request> Future for ResponseFuture<P, S, Request>
 where
-    P: Policy<Request, S::Response, S::Error> + Clone,
+    P: Policy<Request, S::Response, S::Error> + Clone + Unpin,
```

Comment:
Not that I assume `Policy`s will be require to be pinned normally, but I'm curious why this change. Isn't it fine to just project like we did before?

Reply:
Removed in https://github.com/tower-rs/tower/pull/681/commits/416e323024b0f7a21ca545ccee1915cf3c7bd034